readthedocs/analytics/utils.py

```diff
@@ -0,0 +1,107 @@
+"""Utilities related to analytics"""
+
+from __future__ import absolute_import, unicode_literals
+import hashlib
+import logging
+
+from django.conf import settings
+from django.utils.encoding import force_text, force_bytes
+from django.utils.crypto import get_random_string
+import requests
+from user_agents import parse
+
+try:
+    # Python 3.3+ only
+    import ipaddress
+except ImportError:
+    from .vendor import ipaddress
+
+log = logging.getLogger(__name__)   # noqa
+
+# Used to anonymize an IP by zero-ing out the last 2 bytes
+MASK = int('0xFFFFFFFFFFFFFFFFFFFFFFFFFFFF0000', 16)
```

Comment:
Should this just live in the `anonymize_ip_address` function? Not sure if we need it elsewhere.